module/Database/src/Model/SubDecision/OrganRegulation.php

```diff
@@ -0,0 +1,238 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Model\SubDecision;
+
+use Application\Model\Enums\OrganTypes;
+use Database\Model\Member;
+use Database\Model\SubDecision;
+use DateTime;
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\JoinColumn;
+use Doctrine\ORM\Mapping\ManyToOne;
+use IntlDateFormatter;
+
+use function date_default_timezone_get;
+use function str_replace;
+
+#[Entity]
+class OrganRegulation extends SubDecision
+{
+    /**
+     * OrganRegulation author.
+     */
+    #[ManyToOne(targetEntity: Member::class)]
+    #[JoinColumn(
+        name: 'lidnr',
+        referencedColumnName: 'lidnr',
+        nullable: true,
+    )]
+    protected ?Member $author = null;
+
+    /**
+     * Name of the organ.
+     */
+    #[Column(type: 'string')]
+    protected string $name;
+
+    /**
+     * Type of the organ.
+     */
+    #[Column(
+        type: 'string',
+        enumType: OrganTypes::class,
+    )]
+    protected OrganTypes $organType;
```

Comment:
This is part of the model and the form, however, it is never used. Adding this to the actual decision text is possible, but I would refrain from that as that will mean that we cannot convert (almost) all of the old decisions that are currently `other`.